crates/parser2/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    #[cfg(test)]
+    println!("cargo:rerun-if-changed=./std");
+}
```

Comment:
Remove this file? Changing a file in test_files seems to rebuild automatically, so I guess it's not needed.

Reply:
Oops, removed